src/gobupload/relations.py

```diff
@@ -0,0 +1,149 @@
+'''
+Build relations for a specific collection from a catalog.
+
+We retrieve all relations for each source and create the relation on the
+_id field.
+
+'''
+from sqlalchemy import cast
+from sqlalchemy.types import Text
+
+from gobcore.model import GOBModel
+from gobcore.sources import GOBSources
+
+from gobupload.models.model import models
+from gobupload.storage.handler import GOBStorageHandler
+
+model = GOBModel()
+sources = GOBSources()
+storage = None
+
+
+def build_relations(catalog_name, collection_name):
+    """Build all relations for a collection from a catalog.
+
+    :param catalog_name: the name of the catalog
+    :param collection_name: the name of the collection
+    :return:
+    """
+    global storage
+
+    storage = GOBStorageHandler()
+
```

Comment:
Why do you put the initialization here? The other two global variables model and sources are initialized by the module.

Reply:
The GOBStorageHandler makes a connection to the database. By initialising the handler on the module mocking the handler is not possible and tests fail because there's no connection to the database